pandas/core/reshape/merge.py

```diff
@@ -159,9 +159,13 @@ def merge_ordered(left, right, on=None,
         left DataFrame
     fill_method : {'ffill', None}, default None
         Interpolation method for data
-    suffixes : 2-length sequence (tuple, list, ...)
-        Suffix to apply to overlapping column names in the left and right
-        side, respectively
+    suffixes : Sequence
```

Comment:
the docstring will also need the default values included

Reply:
oh, yeah, added! thx for the carefulness!! @simonjayhawkins